commands/cli.go

```diff
@@ -93,19 +97,27 @@ func awsNuke(c *cli.Context) error {
 		}
 	}
 
-	color := color.New(color.FgHiRed, color.Bold)
-	color.Println("\nTHE NEXT STEPS ARE DESTRUCTIVE AND COMPLETELY IRREVERSIBLE, PROCEED WITH CAUTION!!!")
+	if !c.Bool("force") {
+		color := color.New(color.FgHiRed, color.Bold)
+		color.Println("\nTHE NEXT STEPS ARE DESTRUCTIVE AND COMPLETELY IRREVERSIBLE, PROCEED WITH CAUTION!!!")
 
-	prompt := "\nAre you sure you want to nuke all listed resources? Enter 'nuke' to confirm: "
-	shellOptions := shell.ShellOptions{Logger: logging.Logger}
-	input, err := shell.PromptUserForInput(prompt, &shellOptions)
+		prompt := "\nAre you sure you want to nuke all listed resources? Enter 'nuke' to confirm: "
+		shellOptions := shell.ShellOptions{Logger: logging.Logger}
+		input, err := shell.PromptUserForInput(prompt, &shellOptions)
 
-	if err != nil {
-		return errors.WithStackTrace(err)
-	}
+		if err != nil {
+			return errors.WithStackTrace(err)
+		}
 
-	if strings.ToLower(input) == "nuke" {
-		aws.NukeAllResources(account, regions)
+		if strings.ToLower(input) == "nuke" {
+			if err := aws.NukeAllResources(account, regions); err != nil {
+				return err
+			}
+		}
+	} else {
+		if err := aws.NukeAllResources(account, regions); err != nil {
```

Comment:
Add a log statement here that says "The --force flag is set, so proceeding without confirmation."